src/app/static/src/app/models/project.ts

```diff
@@ -1,12 +1,16 @@
+import {DynamicFormMeta} from "@reside-ic/vue-dynamic-form";
+
 export interface Region {
     name: string
     url: string
+    baselineOptions: DynamicFormMeta
 }
 
 export class Region {
-    constructor(name: string, parent: Project) {
-        this.name = name
-        this.url = `/projects/${parent.name}/regions/${name}`.replace(/\s/g, "-").toLowerCase()
+    constructor(name: string, parent: Project, baselineOptions: DynamicFormMeta) {
+        this.name = name;
+        this.url = `/projects/${parent.name}/regions/${name}`.replace(/\s/g, "-").toLowerCase();
+        this.baselineOptions = {...baselineOptions}; //Make our own copy
```

Comment:
this won't be a deep copy, which I think will be a problem

Reply:
Of course! :facepalm: